CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Next Release
 -------------
 
+1.7.0
```

Comment:
I think this is actually a "bugfix" release because its passive and not pushing any new feature.

Reply:
Updated.